cli/hzn.go

```diff
@@ -70,6 +72,32 @@ func main() {
 	// Command flags and args - see https://github.com/alecthomas/kingpin
 	app := kingpin.New("hzn", msgPrinter.Sprintf(`Command line interface for Horizon agent. Most of the sub-commands use the Horizon Agent API at the default location http://localhost (see environment Environment Variables section to override this).
 
+Subcommands Description:
```

Comment:
Can you find a way to display this list and the list in the "Commands:" section in alphabetic order?

Reply:
yes I can do that @dabooz